extensions/python/src/test/java/com/hazelcast/jet/python/PythonInitCleanupTest.java

```diff
@@ -58,7 +59,9 @@
 
     @BeforeClass
     public static void beforeClass() {
-        initialize(2, null);
+        Config config = smallInstanceConfig();
+        config.getJetConfig().setResourceUploadEnabled(true);
```

Comment:
The same piece of code is in `PythonServiceTest` and elsewhere, maybe we can have `smallInstanceWithResourceUploadConfig()`.

Reply:
Addressed :)